pkg/load/load.go

```diff
@@ -186,22 +234,42 @@ func Config(path, unresolvedPath, registryPath string, info *ResolverInfo) (*api
 	return &configSpec, nil
 }
 
-func configFromResolver(info *ResolverInfo) (*api.ReleaseBuildConfiguration, error) {
+func configFromResolver(info *ResolverInfo, options configLoadOpt) (*api.ReleaseBuildConfiguration, error) {
+	queryParams := map[string]string{
+		"org":    info.Org,
```

Comment:
surely we have constants for those query parameters?

Reply:
I guess, but the code here did not use them.